packages/toolpad-app/src/toolpad/Home/index.tsx

```diff
@@ -124,7 +132,7 @@ function CreateAppDialog({ onClose, ...props }: CreateAppDialogProps) {
               label="Name"
               value={name}
               error={createAppMutation.isError}
-              helperText={createAppMutation.isError ? `An app named "${name}" already exists` : ''}
+              helperText={(createAppMutation.error as Error)?.message || ''}
```

Comment:
```suggestion
              helperText={(createAppMutation.isError ? (createAppMutation.error as Error)?.message : ''}
```

Reply:
does this have any particular advantage? 
i thought that checking just for the error would be correct, is there any case where that could have issues?